composio/local_tools/ragtool/actions/rag_query.py

```diff
@@ -25,7 +25,7 @@ class RagToolQuery(Action):
     _tags = ["Knowledge Base"]
     _tool_name = "ragtool"
 
-    def execute(self, request: RagToolQueryRequest, authorisation_data: dict = None):
+    def execute(self, request: RagToolQueryRequest, authorisation_data: dict) -> dict:
```

Comment:
```suggestion
    def execute(self, request_data: RagToolQueryRequest, authorisation_data: dict) -> dict:
```

https://github.com/ComposioHQ/composio/blob/42debbddf3cb2f0e7d77f3f6eaff2f9ea9495a2e/composio/core/local/action.py#L81